<!---->
<launch>
  
	<node pkg="ball_detection" type="ball_detection" name="tennis_ball_detector" output="screen">
		<param name="image_raw" value="/usb_cam/image_raw"/>
		<param name="image_out1" value="/test3"/>
		<param name="image_out2" value="/test4"/>
		<param name="minRad" value="30" />
		<param name="maxRad" value="60" />
		<param name="CamHight" value="0.66" />
		<param name="CamAoV" value="40" />
		<param name="CalibAreaX" value="246" />
		<param name="CalibAreaY" value="420" />
		<param name="CalibAreaHight" value="45" />
		<param name="CalibAreaWidth" value="45" />
		<param name="DetectionAreaHight" value="400" />
		<param name="minKnownTime" value="0" />
		<param name="HoughGradient" value="1" />
		<param name="HoughCanny" value="200" />
		<param name="HoughCenter" value="100" />
		<param name="GaussianSigmaX" value="10" />
		<param name="GaussianSigmaY" value="10" />
		<param name="GaussianSize" value="5" />
		<param name="Areas_Hight" value="/250" />
		<param name="balldetect_out" value="/obstacle_detected" />
		<param name="ballInAreaL_out" value="/nothing" />
		<param name="ballInAreaM_out" value="/nothing" />
		<param name="ballInAreaR_out" value="/nothing" />

	</node>

</launch>
